packages/core/contracts/financial-templates/common/FundingRateApplier.sol

```diff
@@ -39,9 +39,6 @@ abstract contract FundingRateApplier is Testable, Lockable {
     // Last time the `cumulativeFundingRateMultiplier` was updated.
     uint256 public lastUpdateTime;
 
-    // Identifier in funding rate store to query for.
-    bytes32 public fundingRateIdentifier;
```

Comment:
This is no longer used in this contract to fetch a funding rate, instead the `address(this)` is the new key.

Reply:
+1 very nice